src/qsub.jl

```diff
@@ -68,7 +70,7 @@ function launch(manager::Union{PBSManager, SGEManager}, params::Dict, instances_
 end
 
 function manage(manager::Union{PBSManager, SGEManager}, id::Integer, config::WorkerConfig, op::Symbol)
-    if op == :finalize
+    if op == :interrupt
```

Comment:
`:interrupt` is for Ctrl-C, i.e., SIGINT. In managers.jl it sends a `kill -2` which is the command line way of sending a SIGINT to a process.

Don't do anything with `:finalize`

In your implementation of `kill` do the following:

```
remotecall(exit, id)
close(config.io)
kill(config.userdata[:process])
if isfile(config.userdata[:iofile]
    rm(config.userdata[:iofile])
end
```

This ought to work.